identity-server/src/IdentityServer/ResponseHandling/Default/DiscoveryResponseGenerator.cs

```diff
@@ -393,7 +393,7 @@ where scope.ShowInDiscoveryDocument
         // custom entries
         if (!IEnumerableExtensions.IsNullOrEmpty(Options.Discovery.CustomEntries))
         {
-            foreach ((string key, object value) in Options.Discovery.CustomEntries)
+            foreach ((var key, var value) in Options.Discovery.CustomEntries)
```

Comment:
```suggestion
            foreach (var (key, value) in Options.Discovery.CustomEntries)
```

Reply:
Oh that's nice. R# didn't pick up on that.